feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
It's really good that you have decided to add this line of code. Well done!

Reply:
Thanks for the comment!